sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -414,6 +414,7 @@ internal class ViewAnnotationManagerImpl(
     val updatedOptions = options.toBuilder()
       .width(options.width ?: inflatedViewLayout.width)
       .height(options.height ?: inflatedViewLayout.height)
+      .visible(inflatedView.visibility == View.VISIBLE)
```

Comment:
NIT: not part of your fix at all, but in the onGlobalLayoutListener below, looks like we call mapboxMap.updateViewAnnotation once and then a second time if the height needs updating. Maybe we should we refactor the code to make a single call to mapboxMap.updateViewAnnotation, otherwise it makes two calls to the native side.